### PR TITLE
Add support for pod disruption budget

### PR DESCRIPTION
**Pull request by punit-kulal**
Added support for three fields
1. enabled: Default `false` for backward compatibility
2. minAvailable: Default `0` default value doesn't actually impact anything
3. maxUnavailable: No default value.

### CODE REVIEW

File: stable/app/Chart.yaml [hunk 1]
```diff
@@ -15,4 +15,4 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.5.4
+version: 0.5.5
```
Comment (whoAbhishekSah) [inline stable/app/Chart.yaml:18]: Since it's a new feature, shouldn't we bump "minor"?

Comment (punit-kulal) [reply stable/app/Chart.yaml:18]: Previous additions also bumped the patch version. 
I am okay with either.
Will bump to 0.6.0

Comment (whoAbhishekSah) [reply stable/app/Chart.yaml:18]: yeah !

**Comment by whoAbhishekSah** [comment]
@punit-kulal  rebase on main !

**Pull request closed without merging**